ItemList/ItemList/ViewModel.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace ItemList
+{
+    /// <summary>
+    /// This is a class, that represents a ViewModel layer with all fields from View and logic how to handle requests from UI
+    /// </summary>
+    /// <owner>Anton Petrenko</owner>
+    public sealed class ViewModel : INotifyPropertyChanged
+    {
+        /// <summary>
+        /// Stores an instance of RelayCommand to add.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        private RelayCommand addCommand;
+        /// <summary>
+        /// Stores an instance of RelayCommand to delete.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        private RelayCommand deleteCommand;
+        /// <summary>
+        /// Stores instance of selected Phone.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        public Phone selectedItem;
+        /// <summary>
+        /// Initializes addCommand field.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Instance of RelayCommand class.</returns>
+        public RelayCommand AddCommand
+        {
+            get
+            {
+                if (this.addCommand != null)
+                {
+                    return this.addCommand;
+                }
+                this.addCommand = AddNewItemToCollection();
+                return this.addCommand;
+            }
+        }
+        /// <summary>
+        /// Command adds new item to Observable Collection of phones and clean text boxes.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Instance of RelayCommand class.</returns>
+        private RelayCommand AddNewItemToCollection ()
+        {
+            return new RelayCommand(obj =>
+            {
+                this.ListOfItems.Add(new Phone { Manufacture = this.Manufacture, Model = this.Model });
```

Comment:
It is not clear foer me why we are creating new item when the command is initialized.
This code is responcible of the empty line in datagrid that should not exist.
Please remove that.

Reply:
I use this line to to add new item of phone to the list (fields Manufacture and Model compose Phone model), because Add Command should work this way.

Reply:
I'm trying to add a new item to List in this command. Manufacture and Model are filled in UI.
Please advice how to make it right.